ckan/controllers/group.py

```diff
@@ -606,19 +635,19 @@ def delete(self, id):
         try:
             if request.method == 'POST':
                 self._action('group_delete')(context, {'id': id})
-                if self.group_type == 'organization':
-                    h.flash_notice(_('Organization has been deleted.'))
-                else:
-                    h.flash_notice(_('Group has been deleted.'))
-                self._redirect_to(controller='group', action='index')
+                h.flash_notice(_('%s has been deleted.')
+                               % group_type.capitalize())
```

Comment:
> should the parameter be in its own gettext call? if so that's a hidden string for the translators to find. if not the type is not translatable.

Hmm I have an idea.

> Or how about attaching a deleted_flash_notice_message object to the class and let the OrganizationController override it?

If you attach the message to the class, then you can't customize it for a different group_type unless you create a subclass GroupController, which would not normally be necessary for an IGroupForm - as you can see from ckanext/example_igroupform/plugin.py.